src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PreviewWindowModel.cs

```diff
@@ -11,6 +14,49 @@ public class PreviewWindowModel
 
         public string Title { get; private set; }
 
+        public InputGestureCollection CopyGestures;
+
+        public override string ToString()
+        {
+            var sb = new StringBuilder();
+            foreach (var r in PreviewResults)
+            {
+                sb.AppendLine(r.Name);
+                sb.AppendLine("");
```

Comment:
use string.Empty.

Reply:
since it's for a new line, don't pass any string at all `.AppendLine()`

Reply:
Even better :)